metadata_service/proxy/neo4j_proxy.py

```diff
@@ -1158,11 +1159,16 @@ def delete_resource_relation_by_user(self, *,
                                                                       user_key=user_id,
                                                                       id=id
                                                                       )
-
-        delete_query = textwrap.dedent("""
-        MATCH {rel_clause}
-        DELETE rel
-        """.format(rel_clause=rel_clause))
+        if relation_type == UserResourceRel.own:
```

Comment:
could you comment on why we only delete two direction for owning resources?

Reply:
i c, now wouldn't be an issue for bookmark/follow relationships? to do it generic, let's add bi-direction for all three relationships, and do delete for r1, r2 for all three cases? WDYT?

Reply:
Yes it would be good to have consistent relations for follow and bookmark as well. I will update it